src/controllers/midi/midicontroller.cpp

```diff
@@ -288,7 +288,7 @@ void MidiController::processInputMapping(const MidiInputMapping& mapping,
             return;
         }
 
-        QScriptValue function = pEngine->resolveFunction(mapping.control.item);
+        QScriptValue function = pEngine->wrapFunctionCode(mapping.control.item, 5);
```

Comment:
Yes, right. This brach includes already enough improvements to be mergable. Well done.  :-)